exercises/concept/amusement-park/.docs/introduction.md

```diff
@@ -41,7 +41,7 @@ class Airplane
 end
 
 plane = Airplane.new
-plane.wings # returns 2
+plane.number_of_wings # returns 2
```

Comment:
I prefer the `wings` as a method name, rather than `number_of_wings`.  Can we make the change the other way?